ManualText.txt

```diff
@@ -720,7 +717,7 @@ Through the Alertus system, which is the system we use to alert the campus to ha
 •	Give any updated information you gather to the officer but be aware that they cannot “call off” the officer response, the officers will still go even if they say everything is all right.
 •	These alarms will be tested weekly and you will be notified when they are tested.
 •	All panic alarms that are not for testing should generate an incident and log entry with the incident number.
-•	If you receive a panic alarm for the Bennett Center Pool, be aware that the wording on the alert is intended for the Bennett front desk staff and may sound weird to you.  You should call the Bennett front desk as the callback number and dispatch this particular alarm as a medical for a potential drowning.
+•	If you receive a panic alarm for the Bennett Center Pool, be aware that the wording on the alert is intended for the Bennett front desk staff and may sound weird to you. This particular alarm is dispatched as a medical for a potential drowing. You should dispatch the officer first, then call the Bennett front desk as the callback number, and the call Regional. 
 •	You will also receive through this same system, any alerts to the general campus that go to all campus owned computers, Alertus boxes in key strategic locations and announcement monitors (such as in lane and the library).  Examples are in the case of lock down, storm emergencies, school cancellation, security notices such as a dangerous person on campus…etc
```

Comment:
And then conference the bennett front desk to regional if they answer and have information.  Call regional yourself if otherwise.